modules/serverless-ossm-setup.adoc

```diff
@@ -1,12 +1,21 @@
-// Module included in the following assemblies:
-//
-// * serverless/networking/serverless-ossm.adoc
-
 [id="serverless-ossm-setup_{context}"]
-= Using {ProductShortName} with {ServerlessProductName}
+= Integrating {ProductShortName} with {ServerlessProductName}
+
+You can integrate {ProductShortName} with {ServerlessProductName} without using Kourier by completing the following procedure.
+
+.Prerequisites
+
+* You have installed the {ServerlessOperatorName} on your {product-title} cluster.
+* You have installed {ProductName}. {ServerlessProductName} with {ProductShortName} only is supported for use with {ProductName} version 2.0.5 or newer.
+
+[IMPORTANT]
+====
+Do not install the Knative Serving component before completing the following procedures. There are additional steps required when creating the `KnativeServing` custom resource defintion (CRD) to integrate Knative Serving with {ProductShortName}, which are not covered in the general Knative Serving installation procedure of the _Administration guide_.
```

Comment:
s/`KnativeServing` custom resource defintion (CRD)/`KnativeServing` custom resource (CR)

Reply:
CRD is actually correct here afaik - it provides the ability to create other CRs
https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions